packages/manager/apps/web-ongoing-operations/src/pages/update/Update.tsx

```diff
@@ -128,16 +125,11 @@ export default function Update() {
       clearNotifications();
       addSuccess(<OdsText>{t('domain_operations_upload_success')}</OdsText>);
       executeActionOnOperation(operationID);
+      navigate(`${urls.root}${product}`);
```

Comment:
are you not marking the cache as stale for the list of ongoing operations, by invalidating the corresponding query key, so that the table with the ongoing operation is refreshed after your action is done and you are redirected to the ongoing operation table ?